### PR TITLE
Nikhilbio

### PR DESCRIPTION
**Pull request by nruia-penn**
What is the difference between sem joined 2024A, and sem joined 2024F

### CODE REVIEW

**Comment by Clue88** [comment]
hey nikhil, there's no such thing as 2024F in our system:

A means spring
B means summer
C means fall

so you'll have to list your semester as 2024C

**Review by Clue88 — CHANGES_REQUESTED** [review]
see comment

**Comment by nruia-penn** [comment]
I updated it to 2024C .

**Review by nruia-penn — COMMENTED** [review]
testing if this will automatically run tests -- didn't work how do I like tell github that I did the requested changes

File: src/json/members/nikhil.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Nikhil Ruia",
+    "pennkey": "nruia",
+    "major": "Computer Science",
+    "school": "SEAS",
+    "bio": "Hi! My hobbies include cars, movies, and karaoke with friends. I recently started cooking too, but that's not going as well. I am also involved in UPMUNC.",
+    "hometown": "Glenview, IL",
+    "team": "Penn Courses",
+    "roles": ["Backend Developer"],
+    "photo": "https://imgur.com/a/aMYgh9l",
```
Comment (Clue88) [inline src/json/members/nikhil.json]: looks like this photo is the problem, you'll need to use the link to the actual image file (should end with .jpg), not the link to the imgur page

**Review by Clue88 — CHANGES_REQUESTED** [review]

**Review by Clue88 — APPROVED** [review]

**Pull request merged**